gravatar-ui/src/main/java/com/gravatar/ui/components/atomic/Avatar.kt

```diff
@@ -65,6 +91,57 @@ private fun Avatar(model: Any?, size: Dp, modifier: Modifier) {
  * @param modifier Composable modifier
  * @param avatarQueryOptions Options to customize the avatar query
  */
+@Deprecated(
+    "This class is deprecated and will be removed in a future release.",
+    replaceWith = ReplaceWith("com.gravatar.ui.components.atomic.Avatar"),
+    level = DeprecationLevel.WARNING,
+)
+@Composable
+public fun Avatar(
+    state: ComponentState<LegacyProfile>,
+    size: Dp,
+    modifier: Modifier = Modifier,
+    avatarQueryOptions: AvatarQueryOptions? = null,
+) {
+    when (state) {
+        is ComponentState.Loading -> {
+            Box(
+                modifier = modifier
+                    .size(size)
+                    .skeletonEffect(),
+            )
+        }
+
+        is ComponentState.Loaded -> {
+            Avatar(
+                profile = state.loadedValue.toApi2Profile(),
+                size = size,
+                modifier = modifier,
+                avatarQueryOptions = avatarQueryOptions,
+            )
+        }
+
+        ComponentState.Empty -> Avatar(
+            model = if (isNightModeEnabled()) {
+                R.drawable.empty_profile_avatar_dark
+            } else {
+                R.drawable.empty_profile_avatar
+            },
+            size = size,
+            modifier = modifier,
+        )
+    }
```

Comment:
```suggestion
Avatar(state = state.toApi2ComponentStateProfile(), size = size, modifier = modifier, avatarQueryOptions = avatarQueryOptions)
```
I think that would work, right?

Reply:
Yeah, I forgot to update this. Done! Thanks!